apps/gitness/tsconfig.app.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
we dont need three separate tsconfigs.. let's simplify it to just one file

Reply:
Done